2010-09-28  Jie Zhang  <jie@codesourcery.com>

	Backport from mainline:

	gcc/testsuite/
	2010-09-28  Jie Zhang  <jie@codesourcery.com>
	* gcc.dg/Wcxx-compat-12.c: Add -fno-short-enums.

=== modified file 'gcc/testsuite/gcc.dg/Wcxx-compat-12.c'
Index: gcc-4.5/gcc/testsuite/gcc.dg/Wcxx-compat-12.c
===================================================================
--- gcc-4.5.orig/gcc/testsuite/gcc.dg/Wcxx-compat-12.c
+++ gcc-4.5/gcc/testsuite/gcc.dg/Wcxx-compat-12.c
@@ -1,5 +1,5 @@
 /* { dg-do compile } */
-/* { dg-options "-Wc++-compat" } */
+/* { dg-options "-fno-short-enums -Wc++-compat" } */
 
 enum E { A };
 
